bin/run-dev

```diff
@@ -1,9 +1,16 @@
 #! /usr/bin/env bash
 
 # DOC: Run the app locally using Docker. Useful for manual testing.
+# When providing the --prod flag, it uses the production image instead
+# of the dev image. This is useful for testing bugs that only show
+# up in staging/prod. Note that in this mode, it will *not* recompile
+# the app when you make code changes. It will use the version of the code
+# compiled when the prod image was built. Also, because we only build
+# prod for linux/amd64, this will run in emulation on M1 Macs.
```

Comment:
(blocking) Each line should have the `DOC:` part so `bin/help` works correctly.

Probably needs double checking, but I don't believe debugging is available either.